.flowconfig

```diff
@@ -21,6 +21,8 @@
 .*/node_modules/react-popper*
 .*/node_modules/react-hotkeys*
 .*/node_modules/react-select*
+.*/node_modules/create-emotion*
```

Comment:
@gustavo-salazar I tried installing separately but the list goes endlessly after every `npx flow-typed install .....`. I have added in the ignore list. I also noticed that the package `@emotion` is included in the list - https://github.com/ProteinsWebTeam/interpro7-client/blob/master/.flowconfig#L19.

It was the same causing the errors after installing create-emotion.